apprise/plugins/NotifyGoogleChat.py

```diff
@@ -182,11 +200,20 @@ def send(self, body, title='', notify_type=NotifyType.INFO, **kwargs):
             'text': body,
         }
 
-        # Construct Notify URL
+        path = (
+            f'spaces/{self.workspace}/messages?'
+            f'key={self.webhook_key}'
```

Comment:
What if someone wants to use the google chat the old way it was set up and didn't provide a threadkey? will providing one on the path cause it to still work?

Reply:
As far as I have tested, providing a normal gchat URL/URI entail normal functioning of the code...